### DISCUSSION TITLE
last update raudio crashes or becomes very slow and stops playing when adding to the playlist

### DISCUSSION THREAD

**Discussion by githans1** [rern/rAudio#1443] [Bug Reports]
Since the latest update the pi becomes very slow, or even crashes when adding or replacing the playlist with a new another url. going to the playlist is either not possible or takes ages (or a hard reboot) after clicking on it.

Additionally, after first power-on , the root partition of the sd card appears to be no longer readable on my laptop through an usb port. the laptop says that the partition is corrupted.

I did four or five new installs and every time the same happens after one or two adds or replaces to the playlist.

when I for example try to replace and play the station http://capeant.antfarm.co.za:8000/ptafm the sound plays for a few milliseconds then the pi disappears from wifi and comes back with the previously played station.

Does or did anybody have the same problem and how did you solve it?
[console.txt](https://github.com/rern/rAudio/files/12838900/console.txt)

**Comment by R101G** [comment]
Have you tried it with a wired connection? I am using the same model of Pi without this problem. Are there any voltage warnings on the system page? 
I assume you have tried a different SD card.

**Reply by githans1** [reply]
R101G,

Thank you for your reaction.

No I have not tried it with a wired connection. I'll try and see if that makes a difference.

No voltage warnings, only a warning saying stat there was frequency capping.

And yes, I especially for this purpose changed to a new sd card (SanDisk) 

I have a feeling though that it has to do with the read-write procedures. Eg when I take out the card and slip it into my PC, the root partition of card turns out to be inaccessible (also in gparted).
Whereas at the same time inserting it in the pi, it works fine. Until I change the playlist by adding or replacing. Than the pi becomes slow, with intermittend sound (brief sound every 10 seconds or so) and intermittend connectivity.

Leaving the pi off power all night will make it play again without any problems, but 
with the old url. 

I'll try with a wired connection. See if that makes a difference.

**Reply by R101G** [reply]
@githans1 

Are you using a Windows PC? If so, it might have trouble reading the ext4 root partition.

**Reply by githans1** [reply]
I use Linux (Manjaro) to write the image to the sd. Once done, I can access the partitions without any problems. The problems start only after the pi has been booted from the card. Trying to read the sd, root partition, on the PC (dolphin or gparted) after that then turns out not to be possible. Dolphin sais the partition is corrupted. The boot partition is still possible though.

**Reply by R101G** [reply]
Have you tried more than one SD card, rather than just a new one? A fake card might have a problem when the partition is expanded at first boot. That, or a dodgy power supply, are the only things I can think of.

**Reply by githans1** [reply]
Problem solved. I did not get voltage warnings on the system page, but they did appear on "dmesg" together with voltage corrections.

Replacing the RPI power supply with a more powerful one (0.7 A --> 3.0 A) solved the problem. 

Thank you for your suggestions
